mono/metadata/threadpool-ms.c

```diff
@@ -1583,7 +1583,9 @@ void
 ves_icall_System_Threading_ThreadPool_ReportThreadStatus (MonoBoolean is_working)
 {
 	// TODO
-	mono_raise_exception (mono_get_exception_not_implemented (NULL));
+	MonoError error;
+	mono_error_set_generic_error (&error, "System", "NotImplementedException", "");
+	mono_error_set_pending_exception (&error);
```

Comment:
It would be nice to add helper functions to this instead of duplicating these strings.